docs/adr/0004-withdraw-expiry.rst

```diff
@@ -0,0 +1,91 @@
+Withdraw & Expiry
+=================
+
+**Status**: proposed
+**Deciders**: @konradkonrad @palango @hackaugusto @rakanalh
+**Date**: 2019-07-03
+
+Context and Problem Statement
+=============================
+
+Withdraw functionality was introduced into Raiden to enable users to withdraw funds from their open channels into their accounts.
+The implemented flow works as follows:
+
+1. Node (A) initiates the withdraw by dispatching `ActionInitWithdraw` with a monitonic total withdraw value.
+2. Node (A) sends a signed `WithdrawRequest` message to partner to agree on the new total withdraw.
+3. Node (B) receives the withdraw request, validates it and replies with a signed `WithdrawConfirmation` message to node (A).
+4. Node (A) receives the withdraw confirmation, validates it and uses signatures from step (1) and step(2) to send an on-chain transaction
+   to withdraw the funds.
+
+Here, a problem arises when node (B) is unavailable or is not cooperative. In this case, node (A) would have sent a withdraw request,
```

Comment:
Here and everywhere else, perhaps don't repeat node (X|Y|..) here and instead just use the names directly? `A` does something, `B` does something else e.t.c.

Reply:
Changed to Alice / Bob... much better IMO